libbeat/processors/condition_test.go

```diff
@@ -145,6 +145,18 @@ func TestCondition(t *testing.T) {
 			},
 			result: true,
 		},
+		{
+			config: ConditionConfig{
+				HasFields: []string{"proc.cmdline"},
```

Comment:
Could you please add some tests with several fields here? To check all of them should match